jsEngine/main.ts

```diff
@@ -60,6 +60,9 @@ export default class JsEnginePlugin extends Plugin {
 	async loadSettings(): Promise<void> {
 		const loadedSettings = (await this.loadData()) as JsEnginePluginSettings;
 		this.settings = Object.assign({}, JS_ENGINE_DEFAULT_SETTINGS, loadedSettings);
+		if (!this.settings.startupScriptsDirectory) {
+			this.settings.startupScriptsDirectory = normalizePath(this.app.vault.configDir + '/snippets');
```

Comment:
I don't really like the idea of a plugin touching files in `.obsidian`, but maybe it's appropriate here. In any case, I would create a new folder instead of reusing the one for CSS snippets. Maybe `./obsidian/scripts/` is a better option.